util/signal_handler.cc

```diff
@@ -319,6 +343,9 @@ void SignalManager::handle_minidump()
     std::filesystem::copy(log_path, report_dir_path);
   }
 
+  emit_crash_metric(
```

Comment:
If we need to keep this in-line, I'd put it after the clean-up, to make sure that if we trigger an additional fault, we don't prevent directory clean-up.  Since this context runs in the crashed area (I believe?) there's a high chance that we won't be able to create/send the metric, I suspect?

Be interesting to add a hook to be able to generate an actual OTLP-encoded log entry (if they have Splunk logging installed, for instance), where we can parse apart the parameters collected above, and actually generate a remote log entry as well pointing to the host/location/name of the created crash dump info.

Reply:
Ah good point, I will move after clean up.

 Its context is in the newly fork/exec, so I think it would be OK there.

The log entry thing is a cool idea too!  I don't know anything about that bit, so I will need to do some research.